skimage/measure/profile.py

```diff
@@ -16,6 +16,8 @@ def profile_line(img, src, dst, linewidth=1,
         The start point of the scan line.
     dst : 2-tuple of numeric scalar (float or int)
         The end point of the scan line.
+        Note: The final destination point is INCLUDED in the profile, 
+            in constrast to standard numpy indexing.
```

Comment:
Use standard paragraph indentation here. Don't use all-caps. Use `*included*` for emphasis. Remove "Note" and "final".